### DISCUSSION TITLE
Cool Features from other tools

### DISCUSSION THREAD

**Discussion by allisonschiang** [UTDNebula/utd-trends#11] [Ideas]
If anyone has any ideas for features we can integrate, please leave them here and we'll take a look! This includes other school's grade tools, data visualization tools, or anything that might help us build Project Athena!

**Comment by allisonschiang** [comment]
UC Berkeley lets you compare grade data via horizontal bar charts. It might be a good idea to allow comparing sections, but this feels like you're looking at too much data. Also, I like how it compares the course with section average so you can see if its a hard class or relatively hard teacher!
![image](https://user-images.githubusercontent.com/83091926/156447651-32cf3411-b034-442c-af8f-d31c42d454af.png)

**Comment by allisonschiang** [comment]
CMU lets you narrow classes by time of day, as a lot of people don't have the luxury of picking based on time due to having a job or other commitments. If possible, maybe look into a way to narrowing by time first?
![image](https://user-images.githubusercontent.com/83091926/156448457-c1c37d56-3b8d-4586-b508-113b2efe2d6d.png)

**Comment by allisonschiang** [comment]
For accessibility, we can perhaps provide a tutorial for people who use it for the first time. Ex) Cornell's schedule planner
![image](https://user-images.githubusercontent.com/83091926/156448767-05b6c861-f177-4e43-add4-09f60b0df171.png)

**Comment by allisonschiang** [comment]
Might be cool to allow people to customize their project nebula experience if they're gonna be using it a lot, esp during stressful times. ex: custom colored bars
![image](https://user-images.githubusercontent.com/83091926/156451402-077c8594-833d-474f-af77-84de27d26913.png)

**Comment by allisonschiang** [comment]
honestly uwash's grade tool is pretty clean. check it out here: https://uwgrades.com/

**Reply by shoaibhuq** [reply]
Can't access the link at the moment.
